src/templates/BlogIndex.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+import { graphql } from 'gatsby';
+
+import { ArticleLink, Layout, Link, Pagination } from '@components';
+
+export default function BlogIndex(props) {
+  const { currentPage, pageCount } = props.pageContext;
+  const { nodes } = props.data.allFile;
+
+  return (
+    <Layout title={`Blog • Page ${currentPage}`}>
+      <h1>Reactiflux Blog</h1>
+      <div>
+        {nodes.map((post) => (
+          <ArticleLink key={post.name} {...post} />
+        ))}
+        {pageCount > 1 ? (
+          <Pagination>
+            {Array.from({ length: pageCount }, (_, i) => {
+              const page = i + 1;
+              return (
+                <li key={page}>
+                  <Link to={`/blog/${page > 1 ? page : ''}`}>{page}</Link>
```

Comment:
Should this be a `NavLink`?

Reply:
Why?

Reply:
Wouldn't we want the current page to be highlighted?